### ISSUE TITLE
Article filter list not translated.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#866]
```
http://v20120402.html5rocks.appspot.com/ja/tutorials/ for example. See 
screenshot.
```

Original issue reported on code.google.com by `mkwst@google.com` on 3 Apr 2012 at 9:28

Attachments:
- [Screen shot 2012-04-03 at 11.28.29 AM.png](https://storage.googleapis.com/google-code-attachments/html5rocks/issue-866/comment-0/Screen shot 2012-04-03 at 11.28.29 AM.png)

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 12e1b92127fd.
```

Original comment by `mkwst@google.com` on 3 Apr 2012 at 9:54
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 3d6798f8ebd7.
```

Original comment by `mkwst@google.com` on 3 Apr 2012 at 10:16
